R/utils.R

```diff
@@ -338,8 +329,8 @@ insertIntoCallbackMap <- function(map, inputs, output, state, func, clientside_f
                                           clientside_function=clientside_function
                                           )
   if (length(map) >= 2) {
-    ids <- lapply(names(map), function(x) dash:::getIdProps(x)$ids)
-    props <- lapply(names(map), function(x) dash:::getIdProps(x)$props)
+    ids <- lapply(names(map), function(x) getIdProps(x)$ids)
+    props <- lapply(names(map), function(x) getIdProps(x)$props)
```

Comment:
Why the namespace removal?

Reply:
@josegonzalez The `:::` syntax is useful for accessing functions in other packages which are not exported. Usually it's something you'd apply interactively, and not a technique to apply within R package code. 

What's particularly strange is that this was added to access a function within the same package, no idea why I or anyone else did this, but in any event the CRAN check is careful to disallow this behaviour for submitted packages (and with good reason).

Very much a 🙈 moment.